launcher/ui/processlist_unix.cpp

```diff
@@ -127,15 +128,16 @@ ProcDataList processList(const ProcDataList &previous)
     const QStringList procIds = procDir.entryList();
     if (procIds.isEmpty())
         return rc;
-    for (const QString &procId : procIds) {
+    QMutex mutex;
```

Comment:
don't use a mutex, instead actually leverage map:

- let the lambda return ProcData
- return a default-constructed ProcData instead of continue
- filter out all invalid entries once blockingMap is done

Reply:
@milianw can you check again?